distribution/ltp/lite/configs/RHELKT1LITE.20190115

```diff
@@ -1233,7 +1233,8 @@ sysfs04 sysfs04
 sysinfo01 sysinfo01
 sysinfo02 sysinfo02
 
-syslog01 syslog01
+# rsyslog is disabled by default on F30
+#syslog01 syslog01
```

Comment:
Would it be better to check if we're running on Fedora 30 in the knownissue script and exclude the test that way, so we can still run it on the other releases?